src/java.base/linux/classes/jdk/internal/platform/cgroupv1/CgroupV1SubsystemController.java

```diff
@@ -107,7 +107,7 @@ public static long getLongValueMatchingLine(CgroupSubsystemController controller
     }
 
     public static long convertHierachicalLimitLine(String line) {
-        String[] tokens = line.split("\\s");
+        String[] tokens = line.split(" ");
```

Comment:
This delimiter also hasn't changed since the `memory.stat` file was introduced, and since cgroup v1 is in maintenance mode I'd expect it to stay that way.